cit3.0-web/src/components/OpportunityApprovalItem/OpportunityApprovalItem.css

```diff
@@ -0,0 +1,13 @@
+.opportunity-approve-map-container{
+    height: 170px;
+}
+
+.note.note {
+    opacity: 0.7;
+    text-overflow:ellipsis;
+    overflow: hidden;
+    display: -webkit-box !important;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient
https://caniuse.com/mdn-css_properties_box-orient
If I'm reading these correctly, I think we don't want to use -webkit-box-orient on a website that will be production facing. Can we use something else instead, like flexbox?

Reply:
Changed to a JavaScript concern, as this was the only CSS approach that works for multiline I see.

The JS version should support more browsers.